src/java/ognl/DefaultMemberAccess.java

```diff
@@ -47,6 +47,14 @@
  */
 public class DefaultMemberAccess implements MemberAccess
 {
+    /*
+     * Assign an accessibility modification mechanism, based on Major Java Version.
+     *   Note: Can be override using a Java option flag {@link OgnlRuntime#USE_PREJDK9_ACESS_HANDLER}.
+     */
+    private static final AccessibleObjectHandler _accessibleObjectHandler =
+            (OgnlRuntime.isJdk9Plus() && !OgnlRuntime.getUsePreJDK9AccessHandlerValue()) ? new AccessibleObjectHandlerJDK9Plus() :
+            new AccessibleObjectHandlerPreJDK9();
```

Comment:
Would you mind wrapping this whole logic into a method?

Reply:
I tried a compromise in the latest push.  There is now a single method to indicate use of the JDK9Plus handler, combined with the two generator methods.
  
Using that combination in both `OgnlRuntime` and `DefaultMemberAccess`, along with static initializer blocks (which is more in line with other initialization structures in the package), looks a bit cleaner ... I hope. :smile: